src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java

```diff
@@ -223,9 +225,37 @@ public final void setNativeTextBlockLayout(android.text.Layout layout, int leftP
 		_topTextBlockPadding = topPadding;
 	}
 
+	@Override
+	protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec)
+	{
+		super.onMeasure(widthMeasureSpec, heightMeasureSpec);
+		_measureRequested = false;
+	}
+
 	@Override
 	protected void onDraw(android.graphics.Canvas canvas)
 	{
+		if(_measureRequested && !isLayoutRequested())
+		{
+			// Sometimes the isLayoutRequested (PFLAG_FORCE_LAYOUT in Android)
+			// could be reset without any call to onMeasure().
+
+			// The reason for that is unclear and it's only happening in very specific
```

Comment:
It may help to know that `PFLAG_FORCE_LAYOUT` is not reset by the measure, it's reset by Android.Views.View.Layout(). (See the diagram in https://platform.uno/docs/articles/uno-development/Uno-UI-Layouting-Android.html). Could you be in a scenario where requestLayout() is being called after measure has occurred, but just before layout has occurred?

Reply:
I came to a similar conclusion...  but I noticed the onMeasure() is not getting called at all when it's happening.

So the exact cause is not apparent.